docs/architecture/grpc-for-wcf-developers/create-project.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Create a new ASP.NET Core gRPC project - gRPC for WCF Developers
+description: Learn how to create a gRPC project using Visual Studio or from the command line.
+author: markrendle
+ms.date: 09/02/2019
+---
+
+# Create a new ASP.NET Core gRPC project
+
+.NET Core comes with a powerful CLI tool, `dotnet`, which enables you to create and manage projects and solutions from the command line. The tool is closely integrated with Visual Studio, so everything is also available through the familiar GUI interface. This chapter will show both ways to create a new ASP.NET Core gRPC project: first with Visual Studio, then with the .NET Core CLI.
+
+## Create the project using Visual Studio
+
+> [!IMPORTANT]
+> To develop any ASP.NET Core 3.0 app, you need Visual Studio 2019.3 or later with the **ASP.NET and web development** workload installed.
+
+Create an empty solution called **TraderSys** from the *Blank Solution* template. Add a Solution Folder called `src`, then right-click on the folder and choose **Add** > **New Project** from the context menu. Enter `grpc` in the template search box and you should see a project template called `gRPC Service`.
+
+![Add new project dialog showing gRPC Service project template](media/create-project/new-grpc-project.PNG)
```

Comment:
Broken image?

Reply:
Links are obviously case-sensitive on the build system. Fixed.